tests/testthat/test-zzx-deprecated.R

```diff
@@ -1,110 +1,95 @@
+withr::local_options(c(lifecycle_verbosity = "quiet"))
```

Comment:
It's slightly safer to have them inside every `test_that()` call, these calls shouldn't take too much time. Also, `rlang::local_options()` is slightly faster.

Something else: all of these tests can be skipped on CRAN.

Reply:
Scratch that. Let's keep one `local_options()` call at the top of the test file, it's safe enough and easier to read.